access/pagerduty/README.md

```diff
@@ -14,90 +14,98 @@ This guide assumes you have
 - Teleport Enterprise 6.1 or newer with admin permissions and access to `tctl`
 - Pagerduty account already set, with access to creating a new API token.
 
-### Create an access-plugin role and user within Teleport
+## Install the plugin
 
-First off, using an existing Teleport Cluster, we are going to create a new
-Teleport User and Role to access Teleport.
+There are several methods to installing and using the Teleport Pagerduty Plugin:
 
-#### Create User and Role for access.
+1. Use a [precompiled binary](#precompiled-binary)
 
-Log into Teleport Authent Server, this is where you normally run `tctl`. Don't
-change the username and the role name, it should be `access-plugin` for the
-plugin to work correctly.
+2. Use a [docker image](#docker-image)
 
-_Note: if you're using other plugins, you might want to create different users
-and roles for different plugins_.
+3. Install from [source](#building-from-source)
 
+### Precompiled Binary
+
+Get the plugin distribution.
+
+```bash
+$ curl -L https://get.gravitational.com/teleport-access-pagerduty-v7.0.2-linux-amd64-bin.tar.gz
+$ tar -xzf teleport-access-pagerduty-v7.0.2-linux-amd64-bin.tar.gz
+$ cd teleport-access-pagerduty
+$ ./install
 ```
-$ cat > rscs.yaml <<EOF
-kind: user
-metadata:
-  name: access-plugin
-spec:
-  roles: ['access-plugin']
-version: v2
----
+
+### Docker Image
+```bash
+$ docker pull quay.io/gravitational/access-plugin-pagerduty:9.0.2
+```
+
+```bash
+$ docker run quay.io/gravitational/access-plugin-pagerduty:9.0.2 version
+teleport-pagerduty v9.0.2 git:teleport-pagerduty-v9.0.2-0-g9e149895 go1.17.8
+```
+
+For a list of available tags, visit [https://quay.io/](https://quay.io/repository/gravitational/access-plugin-pagerduty?tab=tags)
+
+### Building from source
+
+To build the plugin from source you need Go >= 1.16 and `make`.
+
+```bash
+$ git clone https://github.com/gravitational/teleport-plugins.git
+$ cd teleport-plugins/access/pagerduty
+$ make
+$ ./build/teleport-pagerduty start
+```
+
+## Teleport User and Role
+
+Using Web UI or `tctl` CLI utility, create the role `access-pagerduty` and the user `access-pagerduty` belonging to the role `access-pagerduty`. You may use the following YAML declarations.
+
+### Role
+
+```yaml
 kind: role
 metadata:
-  name: access-plugin
+  name: access-pagerduty
 spec:
   allow:
     rules:
       - resources: ['access_request']
-        verbs: ['list','read']
-      - resources: ['access_plugin_data']
-        verbs: ['update']
-    # if you want to enable auto-approve feature
-    review_requests:
-      roles: ...
-      where: ... 
-version: v3
-EOF
-
-# ...
-$ tctl create -f rscs.yaml
+        verbs: ['list', 'read', 'update']
+version: v4
```

Comment:
Role version should be v5 here and in the other readmes as of `v9`
```suggestion
version: v5
```

Reply:
Fixed in https://github.com/gravitational/teleport-plugins/pull/473/commits/368cc434b1d661e0a63d1f1c66429a1cd0e32b1a